app/src/App.js

```diff
@@ -1,144 +1,145 @@
-                `}
-              >
-                Token Wrapper
-              </h1>
-              <div css="flex-shrink: 0">
-                {wrappedTokenSymbol && (
-                  <Tag mode="identifier">{wrappedTokenSymbol}</Tag>
-                )}
-              </div>
+              Token Wrapper
+            </h1>
+            <div css="flex-shrink: 0">
+              {wrappedToken && wrappedToken.symbol && (
+                <Tag mode="identifier">{wrappedToken.symbol}</Tag>
+              )}
             </div>
-          }
-          secondary={
-            holders && holders.length > 0 ? (
-              <Button
-                mode="strong"
-                label="Wrap tokens"
-                icon={<IconPlus />}
-                onClick={wrapTokensPanel.requestOpen}
-              />
-            ) : (
-              ''
-            )
-          }
-        />
-        <Split
-          primary={
-            holders && holders.length > 0 ? (
-              <Holders
-                holders={holders}
-                onUnwrapTokens={unwrapTokensPanel.requestOpen}
-              />
-            ) : (
-              <NoWrappedTokens isSyncing={isSyncing} />
-            )
-          }
-          secondary={
-            <InfoBox
-              erc20TokenSymbol={erc20TokenSymbol}
-              totalSuply={totalSuply}
-              orgTokenAddress={orgTokenAddress}
-              wrappedTokenAddress={wrappedTokenAddress}
-              wrappedTokenSymbol={wrappedTokenSymbol}
+          </div>
+        }
+        secondary={
+          showHolders && (
+            <Button
+              mode="strong"
+              label="Wrap tokens"
+              icon={<IconPlus />}
+              onClick={wrapTokensPanel.requestOpen}
+              display={layoutName === 'small' ? 'icon' : 'label'}
+            />
+          )
+        }
+      />
+      <Split
+        primary={
+          showHolders ? (
+            <Holders
+              holders={holders}
+              onUnwrapTokens={unwrapTokensPanel.requestOpen}
+              wrappedToken={wrappedToken}
+            />
+          ) : (
+            <NoWrappedTokens
+              isSyncing={isSyncing}
+              onWrapTokens={wrapTokensPanel.requestOpen}
             />
-          }
-        />
+          )
+        }
+        secondary={
+          appStateReady && (
+            <InfoBox outsideToken={outsideToken} wrappedToken={wrappedToken} />
+          )
+        }
+      />
 
-        <Panel
-          panelState={wrapTokensPanel}
-          onAction={actions.wrapTokens}
-          erc20TokenSymbol={erc20TokenSymbol}
-          wrappedTokenSymbol={wrappedTokenSymbol}
-          action="Wrap"
-          info={
-            'You can wrap ' +
-            erc20TokenSymbol +
-            ' into an ERC20-compliant token that you can use within this organization. 1 ' +
-            erc20TokenSymbol +
-            ' = 1 ' +
-            wrappedTokenSymbol
```

Comment:
Nitpick: we generally use template strings (the backtick) rather than string concatenation, or wrap it in an element